international-labour-organization.csl

```diff
@@ -158,12 +160,12 @@
             </group>
           </group>
         </else-if>
-        <else-if type="chapter paper-conference" match="any">
-          <text macro="title" prefix=" " suffix="."/>
+        <else-if type="chapter" match="any">
+          <text macro="title" prefix=" &quot;" suffix="&quot;,"/>
```

Comment:
It's better to use the "quotes" attribute here (`quotes="true"`). See http://docs.citationstyles.org/en/stable/specification.html#quotes